design/ceph/object/store.md

```diff
@@ -73,7 +74,124 @@ If `preservePoolsOnDelete` is set to 'true' the pools used to support the object
   preservePoolsOnDelete: true
 ```
 
-If there is a `zone` section in object-store configuration, then the pool section in the ceph-object-zone resource will be used to define the pools.
+#### Pools shared by multiple CephObjectStore
+
+If user want to use existing pools for metadata and data, the pools must be created before the object store is created. This will be useful if multiple objectstore can share same pools. The detail of pools need to shared in `radosNamespaces` settings in object-store CRD. Now the object stores can consume same pool isolated with different namespaces. User can create via [Pool CRD](/Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md), this is need to present before the object store is created.
+
+```yaml
+  radosNamespaces:
```

Comment:
Please add the spec headers above this so that we have a specific design plan for where in the spec this config lives. Also, please include the current pool config options as a reference (with the yaml commented out) so we see better at a glance how the new configs relates to old ones.